client/platform/desktop/frontend/components/ImportDialog.vue

```diff
@@ -37,10 +46,15 @@ export default defineComponent({
 
     const sortedFpsOptions = computed(() => {
       const filteredOptions = FPSOptions
-        .filter((v) => v <= Math.round(argCopy.value.jsonMeta.originalFps));
-      filteredOptions.push(argCopy.value.jsonMeta.originalFps);
-      filteredOptions.sort((a, b) => a - b);
-      return uniq(filteredOptions);
+        .filter((v) => v.value <= Math.round(argCopy.value.jsonMeta.originalFps));
+      const videoFrameRateIndex = filteredOptions.findIndex((item) => item.value === -1);
+      if (videoFrameRateIndex !== -1) {
+        filteredOptions[videoFrameRateIndex] = {
+          text: `Video Frame Rate (${argCopy.value.jsonMeta.originalFps})`,
```

Comment:
Reverse text and number so number shows first, shorten to `Video FPS`

```
29.97 (Video FPS)
```

That way you see the number instead of this:

![Screenshot from 2021-10-04 15-11-33](https://user-images.githubusercontent.com/4214172/135910265-d0705461-cd0f-46fd-973d-e282a93ed246.png)